src/web/components/ClientSideTokenGeneration/CstgDomainHelper.spec.ts

```diff
@@ -39,17 +39,6 @@ describe('test domain helper functions', () => {
       expect(extractTopLevelDomain(stringsList)).toEqual('test.com');
     });
   });
-  describe('validate domains', () => {
-    it.each(['test.c', 'https://test', '', 't    st.com', 'test'])(
-      'should check the domain is invalid',
-      (stringsList) => {
-        expect(isValidDomain(stringsList)).toEqual(false);
-      }
-    );
-    it.each(validDomainsList)('should return the domain is valid', (stringList) => {
-      expect(isValidDomain(stringList)).toEqual(true);
-    });
-  });
```

Comment:
Because this functionality is done on the backend now, this helper function is no longer needed